assets/src/state/searchPageState.ts

```diff
@@ -6,12 +6,32 @@ import {
   SearchQuery,
 } from "../models/searchQuery"
 import { VehicleId } from "../realtime"
+import { RouteId, RoutePatternId } from "../schedule"
+
+export enum SelectedEntityType {
+  VEHICLE = 1,
+  ROUTE,
+}
```

Comment:
**suggestion:** usually for enum types we've used title case for the values. See [`ConnectionStatus`](https://github.com/mbta/skate/blob/f49857bd10e68fdabd34f4a04cc6c86065ad1949/assets/src/hooks/useSocket.ts#L6) for instance.